packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -871,26 +798,25 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
     }
   }
 
-  private _onItemMouseDown(item: IContextualMenuItem, ev: React.MouseEvent<HTMLElement>) {
-    if (item.onMouseDown) {
+  private _onItemMouseDown = (item: IContextualMenuItem, ev: React.MouseEvent<HTMLElement>): void => {
+    if (item.onMouseDown && this._enterTimerId === undefined) {
```

Comment:
Why was this added, mouseDown should fire against the element it was fired against at the time that is was fired. This makes it to where if you hovered over a different item than the one you mousedDown on quickly enough the mousedown would do nothing and the old hovered over element would become focused